Map/MapManager.gd

```diff
@@ -25,13 +27,16 @@ func create_map(map_floors_width: Array[int]) -> MapBase: ## Generates and Popul
 			var _room_event: EventBase = GlobalVar.EVENTS_CLASSIFICATION[_rand_type_index].new()
 			var _generated_room: RoomBase = RoomBase.new()
 			_generated_room.room_event = _room_event
+			# Test statement to test player room placement, Remove later
+			if (index_width == 4 and index_height == 3):
```

Comment:
we should have a way as dev team to put the player where we want on the map to not have to do that precisely
I'll try to think of a way in the map movement PR

Reply:
I'm thinking of opening up an export variable on the MapUI to force a player position. Since your PR will focus on setting what the player position is, this export I have will just override it so we can use that for testing.

Reply:
seems good